README.md

```diff
@@ -24,9 +24,7 @@ application's ACL (access control list) entries.
 ### Tutorials
 
 - [Getting started with LoopBack](http://docs.strongloop.com/display/LB/Getting+started+with+LoopBack)
```

Comment:
This link should be to http://loopback.io/doc/en/lb3/Getting-started-with-LoopBack.html